src/Codec/Xlsx/Writer.hs

```diff
@@ -120,12 +121,14 @@ singleSheetFiles n cells ws = runST $ do
             , Just . elementListSimple "sheetData" $ sheetDataXml cells (ws ^. wsRowPropertiesMap)
             ] ++
             map (Just . toElement "conditionalFormatting") cfPairs ++
-            [ nonEmptyElListSimple "mergeCells" . map mergeE1 $ ws ^. wsMerges
+            [ Just $ countedElementList "dataValidations" $ map (toElement "dataValidation") dvPairs
```

Comment:
Ah, I didn't realize that. But then it seems like `conditionalFormatting` is also incorrectly placed, no? It should come right before `dataValidations`.